testutils/base.go

```diff
@@ -18,10 +18,17 @@
 package testutils
```

Comment:
Can you check what the go conventions say about the plural here? Maybe it should be `testutil`.